### DISCUSSION TITLE
Employing fast training for RTMDet

### DISCUSSION THREAD

**Discussion by ghost** [open-mmlab/mmdetection#10006] [Q&A] [answered]
Is there any plan to employ fast training for RTMDet in MMDETECTION that is only available in MMYOLO?

We have concerns regarding the GPT-3.0 license used in MMYOLO, hence prefer to use MMDETECTION.

**Comment by RangiLyu** [comment, marked as answer]
If your business only involves the deployment of the model, then don't worry. The RTMDet algorithm itself is still under Apache 2.0 license. Feel free to use it in MMYOLO.

Since some components of the fast training are derived from YOLOv5, these parts of the training process are under the GPL License, and cannot be merged into MMDetection.

**Reply by ghost** [reply]
Just to clarify. 

So, GPL license is only enforced on the source code in MMYOLO, but RTMDet models trained using MMYOLO are not bounded by GPL license. Hence, we can deploy the trained RTMDet models using MMDetection source code and only be affected by Apache 2.0 license. Am I correct?

**Reply by hhaAndroid** [reply]
Yes

**Reply by ghost** [reply]
Understood. Thank you all for the replies and clarification.
